README.rst

```diff
@@ -114,7 +114,7 @@ To do
 
 .. |Build Status| image:: https://travis-ci.org/bigcommerce/bigcommerce-api-python.png?branch=master
```

Comment:
Would you mind changing this to `https://api.travis-ci.org/bigcommerce/bigcommerce-api-python.svg?branch=master` while you're at it?